src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.approved.txt

```diff
@@ -802,9 +802,6 @@ namespace GraphQL.Introspection
     public static class SchemaIntrospection
     {
         public static readonly string IntrospectionQuery;
```

Comment:
Let's move this to test to cut public API surface.

Reply:
Agreed. Lmk where

Reply:
I guess anywhere will do if it’s tests. Not at a PC just now.

Reply:
Anywhere will be OK :)

Reply:
I cannot move it until #1549 is pulled in, as the name converters rely on `nameof(SchemaIntrospection)`.  After #1549 then I can make this change.

Reply:
Done